apps/chrome/chrome.py

```diff
@@ -30,6 +30,16 @@
 """
 
 
+@mod.action_class
+class Actions:
+    def chrome_mod(key: str):
```

Comment:
@rntz I saw you had a concern about this approach here: https://github.com/knausj85/knausj_talon/pull/892#pullrequestreview-1013594471.

It would be good to have a general way to do this, as described by https://github.com/knausj85/knausj_talon/issues/942. If you have any suggestions for this, I'm happy to implement it, otherwise I can just leave it specific to this code for now.